### ISSUE TITLE
MSHNet中卷积替换

### ISSUE THREAD

**Issue by Ws1www** [JN-Yang/PConv-SDloss-Data#7]
您好，请问在MSHNet网络中，如何替换的卷积模块？我是把下采样过程中的第一个卷积以及encoder-decoder中block中的卷积换成了PConv，可是这样好像效果并没有提升，请问有没有什么好的建议？

**Comment by JN-Yang** [comment]
仅改ResNet的前两层和encoder的第一个下采样层。

`
class ResNet(nn.Module):
    def __init__(self, in_channels, out_channels, stride=1, PC=False):
        super(ResNet, self).__init__()
        self.PC = PC
        if PC:
            self.conv1 = PConv(in_channels, out_channels, 4, stride)
            self.conv2 = PConv(out_channels, out_channels, 3, 1)
        else:
            self.conv1 = nn.Conv2d(in_channels, out_channels, kernel_size=3, stride=stride, padding=1)
            self.conv2 = nn.Conv2d(out_channels, out_channels, kernel_size=3, padding=1)
    ...
    def forward(self, x):
        residual = x
        if self.shortcut is not None:
            residual = self.shortcut(x)
        if self.PC:
            out = self.conv1(x)
            out = self.conv2(out)
        else:
            out = self.conv1(x)
            out = self.bn1(out)
            out = self.relu(out)
            out = self.conv2(out)
            out = self.bn2(out)
      ...


class MSHNet(nn.Module):
    def __init__(self, input_channels, block=ResNet, PC=True):
        ...
        self.encoder_0 = self._make_layer(param_channels[0], param_channels[0], block, PC=PC)
        ...
    def _make_layer(self, in_channels, out_channels, block, block_num=1, PC=False):
        layer = []
        layer.append(block(in_channels, out_channels, PC=PC))
        ...

`

**Comment by Ws1www** [comment]
好的非常感谢，对于损失的话，就直接替换掉原来的SLS损失就可以是吗？
